packages/renderer/src/lib/kube/KubeActions.svelte

```diff
@@ -1,5 +1,15 @@
 <script lang="ts">
+import { Button } from '@podman-desktop/ui-svelte';
+
 import KubeApplyYamlButton from '/@/lib/kube/KubeApplyYAMLButton.svelte';
+import { handleNavigation } from '/@/navigation';
+import { NavigationPage } from '/@api/navigation-page';
+
+function navigateToKubernetesResourceCreate(): void {
+  return handleNavigation({ page: NavigationPage.KUBERNETES_RESOURCE_CREATE });
+}
 </script>
 
 <KubeApplyYamlButton />
+<Button title="Create Kubernetes Resource" on:click={navigateToKubernetesResourceCreate}>Create Resource</Button>
```

Comment:
enhancement: I would expect a `<KubeCreateResourceButton>` so we can reuse it somewhere else as well (like it's done for ApplyYaml)

Reply:
Done ✅